data-imports/import-aep.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
+
+import argparse
+import os
+import os.path
+import sys
+import logging
+
+from utils import *
+
+mkdir_p(BASE_DIR)
+
+logging.basicConfig(format='[%(asctime)s] {%(pathname)s:%(lineno)d} %(levelname)s - %(message)s',
+    datefmt='%H:%M:%S',
+    stream=sys.stdout,
+    level=logging.INFO)
+log = logging.getLogger(__name__)
+
+table_name = 'aep_list'
+
+def main(argv):
+    parser = argparse.ArgumentParser(description='Import hpd buildings dataset.')
```

Comment:
yeah, i can clean up the abstractions to lessen this sort of thing in a different PR